src/Semaphore.sol

```diff
@@ -54,6 +63,14 @@ contract Semaphore is IWorldID, SemaphoreCore, Verifier, SemaphoreGroups {
 
     mapping(uint256 => RootHistory) internal rootHistory;
 
+    mapping(uint256 => uint256) public latestRoots;
```

Comment:
Do we need to initialize this in `createGroup`? Currently it is not set untill `memberAdded`.

Reply:
It's not explicitly, but it is in the sense that now, the empty mapping does contain the hardcoded empty leaf value.